packages/cursorless-vscode/src/ide/vscode/VscodeTextEditorImpl.ts

```diff
@@ -173,6 +173,10 @@ export class VscodeTextEditorImpl implements EditableTextEditor {
     await vscode.commands.executeCommand("editor.action.revealDefinition");
   }
 
+  public async revealDefinitionAside(_range?: Range): Promise<void> {
+    await vscode.commands.executeCommand("editor.action.revealDefinitionAside");
```

Comment:
Thinking on it, I think it would be better to have `followLinkAside`, because I haven't used `revealDefinition` in probably a year. Follow link is much more robust. Here is where that is implemented: https://github.com/cursorless-dev/cursorless/blob/main/packages/cursorless-vscode/src/ide/vscode/VscodeOpenLink.ts

You should be able to just do a case-matched find/replace on your PR to change the name, then there will be adding the options object and opening aside in VscodeOpenLink.ts

We could pass an options object indicating to open aside, as suggested in another of my comments

Reply:
Same here; I don't use reveal definition at all anymore. Almost like we should deprecate that one?

Reply:
Yeah I don't think there are any examples where it is better than follow link